docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3019,19 +3021,23 @@ Limitations:
 
 Many people choose to use nicknames instead of their "real" names. A common example is an 
 Alexander who chooses to go by Alex. These individuals might write their nicknames on forms 
-which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/carltonnorthern/nicknames/blob/master/names.csv>`_. 
+which should be recorded. Here is a list of 1080 names and their `relevant nicknames <https://github.com/ihmeuw/vivarium_research_prl/blob/main/noise/nicknames.csv>`_. 
```

Comment:
Oops sorry for the back and forth, but we will need to still cite where we got this.

Reply:
Hahah sure, makes sense